maintainers/maintainer-list.nix

```diff
@@ -8285,6 +8285,12 @@
     name = "Fausto Núñez Alberro";
     keys = [ { fingerprint = "668E 01D1 B129 3F42 0A0F  933A C880 6451 94A2 D562"; } ];
   };
+  folliehiyuki = {
+    email = "folliekazetani@protonmail.com";
+    github = "folliehiyuki";
+    githubId = 67634026;
+    name = "Hoang Nguyen";
+  };
```

Comment:
this will need to be a separate commit, please. It also needs to be the first commit in the PR.